src/components/FullScreenMedia.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import {
+    Modal,
+    TouchableOpacity,
+    View,
+    Dimensions
+} from 'react-native'
+import FontedText from './common/FontedText'
+import EStyleSheet from 'react-native-extended-stylesheet'
+import ZoomableMedia from './ZoomableImage'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import PropTypes from 'prop-types'
+import Video from 'react-native-video'
+
+
+class FullScreenMedia extends React.Component {
+    render() {
+        function displayWithRequisiteComponent(uri, handlePress) {
+            if (uri.slice(uri.length - 4).match('.mp4')) {
+                return <Video
+                    source={{uri: uri}}
+                    style={{
+                        width: Dimensions.get('window').width,
+                        height: Dimensions.get('window').height
+                    }}
+                    controls={true}
+                    repeat={true}
```

Comment:
The video behavior on the Zooniverse web app is for video not to repeat. I think there are accessibility reasons, but I'm having difficulty finding a source. At minimum for consistency I lean towards not repeating, but don't have a strong opinion, defer to you.

Reply:
Totally. The reason for repeating in this context is that this library offers minimal control over playback for users, and these videos are short enough that it seems likely that the volunteer will not catch the action the first time. So I'd like to leave this for mobile for now.